webapp/src/styles/_components.scss

```diff
@@ -59,3 +59,13 @@
         color: var(--error-text);
     }
 }
+
+// Do not modify this class, this is a direct class from mattermost-webapp
+// This class is used for the dimensions of author_icon in Mattermost slack attachment
+.post .attachment {
+    // To make sure that this CSS does not conflict with the CSS of other plugins
+    img[src*="mattermost-plugin-azure-devops/static/repos-icon.svg"], img[src*="mattermost-plugin-azure-devops/static/boards-icon.svg"] {
+        width: 20px;
+        height: 20px;
+    }
+}
```

Comment:
same this is also not synced with #87

Reply:
I will take a pull when any one of the PR is merged

Reply:
synced